source/security/incidents.html.md

```diff
@@ -0,0 +1,124 @@
+
+### Examples of incidents
+
+- All or part of the design system website being unavailable
+- The design system website being defaced
+- One or more of our platform accounts being compromised eg: npm, mailchimp, bitwarden
+- Malicious code being shipped in govuk-frontend
+- Linking to a malicious resource in our documentation eg: a port of govuk-frontend that contains malicious code
+- A serious accessibility bug is shipped/discovered in govuk-frontend
+
+### Examples of things that are not incidents
+
+Note that it's still valuable to fix issues like these ones promptly but they don't need to be treated as incidents as they don't seriously impact the design system service.
+
+- Inaccurate information on the design system website
+- A non-blocking bug in govuk-frontend code
+- An accessibility bug in govuk-frontend that doesn't block users from using services
+- A user using the design system inappropriately
+- A user needing technical support with the design system or govuk-frontend
+
+## Responding to an incident
+
+Once something has been identified as an incident, the most valuable thing to do is attempt to resolve it. However, it's important to follow procedure so that we can properly document and manage incidents during and after they happen.
+
+Incidents take priority over all other work.
+
+### Let the team know
+
+The first thing to do is let the team know an incident is happening. Post a message with a summary of the incident in progress. Use '@here' so that the whole team is aware.
+
+### Establish an incident team
+
+You should assign 2 roles to help manage the incident:
+
+1. An incident lead. This is someone who will coordinate the incident from discovery to resolution. It's recommended to pick an incident lead based on expertise, availability and knowledge of the problem.
+2. A comms lead. This person will be responsible for communicating the incident and its status to the team, across our community channels and to key stakeholders.
+
+Incidents are varied and can require different roles to resolve. You also may discover exactly which roles you need as the incident progresses. It's recommended to request roles on the team as you need them.
+
+### Start an incident report
+
+The incident team at this point should start an incident report so that they can keep track of the incident, it's impact, it's severity, a timeline of events and, once resolved, a list of actions taken out of an incident review. You can find a template for incident reports, as well as a record of all previous incidents by year, [in the INCIDENTS folder in the team drive](https://drive.google.com/drive/u/0/folders/1plXDUGFIoyzWubWNC9J_IjMiEN8eh9pO).
+
+The nature of incidents means that you may not have time to keep the report up to date as the incident is happening. If the incident team are unable to do this, you should update the incident report after it's been resolved.
```

Comment:
```suggestion
The nature of incidents means that you may not have time to keep the report up to date as the incident is happening. If the incident team are unable to do this, you should update the incident report as soon as possible after it's been resolved.
```

Reply:
Aha, so originally I had this in, but the first sentence of the next section, immediately at the end of this sentence, reads:

>You should decide on a severity score for your incident as soon as possible. We have 4 levels of incident severity:

It feels a bit clunky to me having 'as soon as possible' followed by 'as soon as possible' again. Maybe that's actually fine? The secret agenda of these docs is to get folks to remember ot actually fill out and use the incident reports and not just forget about them, so I'd rather **order** folks to do it than be gentle. What do you reckon?

Reply:
```suggestion
The nature of incidents means that you may not have time to keep the report up to date as the incident is happening. If the incident team are unable to do this, you should update the incident at the earliest opportunity.
```

Maybe?